runtime/tests/interpreter/string_test.go

```diff
@@ -172,6 +173,62 @@ func TestInterpretStringEncodeHex(t *testing.T) {
 	)
 }
 
+func TestInterpretStringFromUtf8(t *testing.T) {
+	t.Parallel()
+
+	type Testcase struct {
+		expr     string
+		expected any
+	}
+
+	testCases := [...]Testcase{
+		// String.fromUTF(str.utf8) = str
+		{`"omae wa mou shindeiru".utf8`, "omae wa mou shindeiru"},
+		{`"would you still use cadence if i was a worm 🥺😳 👉👈".utf8`, "would you still use cadence if i was a worm 🥺😳 👉👈"},
```

Comment:
subjunctive
```suggestion
		{`"would you still use cadence if i were a worm 🥺😳 👉👈".utf8`, "would you still use cadence if i were a worm 🥺😳 👉👈"},
```

Reply:
but the meme uses the indicative

https://knowyourmeme.com/memes/would-you-still-love-me-if-i-was-a-worm